tools/setup/install-sgrep

```diff
@@ -1,23 +0,0 @@
-#!/usr/bin/env bash
-set -e
-
-version=0.4.9b5
-tarball=sgrep-$version-ubuntu-16.04.tgz
-sha256=9e57323fd0eb9133b7ff301a6be8361c073c3bfe6e6959ca1b622e5abc176e03
-tarball_url=https://github.com/returntocorp/sgrep/releases/download/v$version/$tarball
-
-check_version () {
-    out="$(sgrep-lint --version 2>/dev/null)" && [ "$out" = "$version" ]
-}
-
-if ! check_version; then
-    tmpdir="$(mktemp -d)"
-    trap 'rm -r "$tmpdir"' EXIT
-    cd "$tmpdir"
-    wget -nv "$tarball_url"
-    sha256sum -c <<< "$sha256  $tarball"
-    tar -xzf "$tarball" -C /usr/local/lib/
-    ln -sf /usr/local/lib/sgrep-lint-files/sgrep-lint /usr/local/bin/sgrep-lint
-    ln -sf /usr/local/lib/sgrep-lint-files/sgrep /usr/local/bin/sgrep
-    check_version
```

Comment:
Also this second `check_version` was intentional and shouldn’t have been deleted (#14695).